packages/composites/dns-record-check/schemas/env.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$id": "https://external-adapters.chainlinklabs.com/schemas/dns-record-check-adapter.json",
+  "title": "@chainlink/dns-record-check-adapter env var schema",
```

Comment:
This one imports and uses `source/dns-query`, it needs a required env var from there: `DNS_PROVIDER`